doc/source/cluster/kubernetes/user-guides/observability.md

```diff
@@ -21,13 +21,42 @@ kubectl logs $KUBERAY_OPERATOR_POD -n $YOUR_NAMESPACE | tee operator-log
+
+
+#### RayCluster Status State List
+
+| State     | Description                                                                                                                            |
+|-----------|----------------------------------------------------------------------------------------------------------------------------------------|
+| Ready     | The state will be set to Ready once all the Pods in the cluster are ready. The State will remain Ready until the cluster is suspended. |
+| Suspended | The state will be set to Suspended when `Spec.Suspend = true` and all the Pods in the cluster are deleted.                             |
+
+#### RayCluster Status Condition List
+
+Although `Status.State` can represent the cluster situation, it is still only a single field. By enabling the feature gate `RayClusterStatusConditions` on the KubeRay v1.2.1, you can access to new `Status.Conditions` for more detailed cluster history and states.
+
+| Type                     | Status | Reason                         | Description                                                                                                          |
+|--------------------------|--------|--------------------------------|----------------------------------------------------------------------------------------------------------------------|
+| RayClusterProvisioned    | True   | AllPodRunningAndReadyFirstTime | Once all the Pods in the cluster are ready, this condition will be True and remain True even if some Pods die later. |
+|                          | False  | RayClusterPodsProvisioning     |                                                                                                                      |
+| RayClusterReplicaFailure | True   | FailedDeleteAllPods            | This condition will be set when there is a reconciliation error, otherwise the condition will be cleared.            |
+|                          | True   | FailedDeleteHeadPod            | Please refer to the Reason and the Message of the condition for more detailed debugging information.                 |
+|                          | True   | FailedCreateHeadPod            |                                                                                                                      |
+|                          | True   | FailedDeleteWorkerPod          |                                                                                                                      |
+|                          | True   | FailedCreateWorkerPod          |                                                                                                                      |
+| HeadPodReady             | True   | HeadPodRunningAndReady         | This condition will be True only the HeadPod is currently ready, otherwise it will be False.                         |
```

Comment:
```suggestion
| HeadPodReady             | True   | HeadPodRunningAndReady         | This condition will be True only if the HeadPod is currently ready; otherwise, it will be False.                         |
```